pavelib/utils/test/suites/nose_suite.py

```diff
@@ -113,25 +122,47 @@ def __init__(self, *args, **kwargs):
         self.test_id = kwargs.get('test_id', self._default_test_id)
         self.fasttest = kwargs.get('fasttest', False)
 
+        self.processes = kwargs.get('processes', None)
+        self.randomize = kwargs.get('randomize', None)
+
+        if self.processes is None:
+            self.processes = -1 if self.root == 'lms' else 0
```

Comment:
`-1` means "use as many processes as cores", right? Could you add a comment explaining the reasoning for this section? It looks kinda magical otherwise.